src/error.rs

```diff
@@ -470,6 +475,12 @@ impl<'src> ColorDisplay for Error<'src> {
       UnstableFeature { unstable_feature } => {
         write!(f, "{unstable_feature} Invoke `just` with `--unstable`, set the `JUST_UNSTABLE` environment variable, or add `set unstable` to your `justfile` to enable unstable features.")?;
       }
+      WorkingDirectoryIo { recipe, working_directory, io_error } => {
+        match io_error.kind() {
```

Comment:
What does the error message for `NotFound` look like without this special case? If it's reasonable, i.e., it's clear that the directory doesn't exist, then we could remove this special case.

Reply:
On linux it's `No such file or directory`, which is indeed clear enough